### PR TITLE
Add flokk to samples and codelabs

### PR DESCRIPTION
**Pull request by domesticmouse**
Adding gskinner's flokk desktop app to the codelabs and sample apps section.

### CODE REVIEW

File: src/desktop.md [hunk 1]
```diff
@@ -587,13 +587,19 @@ You can run the following samples as desktop apps,
...
+Flutter Gallery [running web app][], [repo][flutter-gallery-repo]
 : A samples project hosted on GitHub to help developers
   evaluate and use Flutter. The Gallery consists of a
   collection of Material design widgets, behaviors,
   and vignettes implemented with Flutter.
   You can clone the project and run Gallery as a desktop app
   by following the instructions provided in the [README][].
+  
+Flokk [annonucement blogpost][gskinner-flokk-blogpost], [repo][gskinner-flokk-repo]
+: A fresh and modern Google Contacts manager that integrates with GitHub and Twitter
```
Comment (domesticmouse) [inline src/desktop.md:599]: @csells can you word smith this into something less sales-y?

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Can we make this more factual and less marketing-ish?

**Comment by sfshaza2** [comment]
@csells, can you review this asap? Otherwise I can't accept it before freezing the website repo tomorrow morning for some planned updates. thx

File: src/desktop.md [hunk 2]
```diff
@@ -587,13 +587,19 @@ You can run the following samples as desktop apps,
...
+  by our friends at [gskinner][]. It syncs with your Google account, imports all your 
+  Contacts and allows you to manage them. Additionally, you can link Twitter and 
+  Github accounts for each contact, and see all of their social activities at a glance.
```
Comment (sfshaza2) [inline src/desktop.md:602]:
```suggestion
Flokk [announcement blogpost][gskinner-flokk-blogpost], [repo][gskinner-flokk-repo]
: A Google contacts manager that integrates with GitHub and Twitter. 
  It syncs with your Google account, imports your contacts,
  and allows you to manage them.
```

File: src/desktop.md [hunk 3]
```diff
@@ -605,6 +611,9 @@ Flutter Gallery [running web app][], [repo][]
 [`menubar`]: {{site.github}}/google/flutter-desktop-embedding/tree/master/plugins/menubar
 [Photo Search app]: {{site.repo.organization}}/samples/tree/master/experimental/desktop_photo_search
 [README]: {{site.github}}/flutter/gallery#flutter-gallery
-[repo]: {{site.github}}/flutter/flutter/tree/master/dev/integration_tests/flutter_gallery
+[flutter-gallery-repo]: {{site.github}}/flutter/flutter/tree/master/dev/integration_tests/flutter_gallery
+[gskinner]: https://www.gskinner.com/
+[gskinner-flokk-repo]: https://github.com/gskinnerTeam/flokk
+[gskinner-flokk-blogpost]: https://blog.gskinner.com/archives/2020/09/flokk-how-we-built-a-desktop-app-using-flutter.html
```
Comment (sfshaza2) [inline src/desktop.md:617]:
```suggestion
[flutter-gallery-repo]: {{site.github}}/flutter/flutter/tree/master/dev/integration_tests/flutter_gallery
[gskinner-flokk-repo]: {{site.github}}/gskinnerTeam/flokk
[gskinner-flokk-blogpost]: https://blog.gskinner.com/archives/2020/09/flokk-how-we-built-a-desktop-app-using-flutter.html
```

**Review by sfshaza2 — APPROVED** [review]
@domesticmouse, I think Chris has too much going on, but I am ok with my suggested text.  :)

**Pull request merged**